recipes/r-rcppensmallen/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set version = '0.2.14.2.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rcppensmallen
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/RcppEnsmallen_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/RcppEnsmallen/RcppEnsmallen_{{ version }}.tar.gz
+  sha256: 6550dd0b198abb210e1a0c148dc0f6631a7b39424fe7a2ed94e02b6ba5780393
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - _openmp_mutex                # [linux]
```

Comment:
the mutex should come automatically via run_exports ... can you try removing it here?

Reply:
```suggestion
```

Reply:
@dbast from the CI, removing this line caused the linux build to fail. Would you like me to re-add it?